### PR TITLE
fix: update lock

### PR DESCRIPTION
**Pull request by greged93**
# Pull Request type

Time spent on this PR: 0.1 day

Resolves: #NA

Please check the type of change your PR introduces:

- [x] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no API changes)
- [ ] Build-related changes
- [ ] Documentation content changes
- [ ] Testing

## What is the new behavior?
Updates the lock to point to the new commits of the cairo vm and blockifier forks.

## Does this introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: crates/ef-testing/src/models/case.rs [hunk 1]
```diff
@@ -156,7 +156,6 @@ impl BlockchainTestCase {
             ));
         }
         let gas_price = gas_price | effective_gas_price;
-        let transaction_cost = gas_price * expected_gas_used;
```
Comment (Eikix) [inline crates/ef-testing/src/models/case.rs:159]: we don't need this anymore ?

**Review by Eikix — COMMENTED** [review]
one question

File: crates/ef-testing/src/models/case.rs (hunk 1 above)
Comment (greged93) [reply crates/ef-testing/src/models/case.rs:159]: No this isn't used anymore, was removed in a PR by eni

**Review by Eikix — APPROVED** [review]
lgtm

**Pull request merged**